_studio/shared/umc/codec/h265_dec/src/umc_h265_va_packer.cpp

```diff
@@ -378,7 +378,7 @@ void PackerVA::CreateSliceDataBuffer(H265DecoderFrameInfo const* sliceInfo)
         size += NalUnitSize + 3;
     }
 
-    AlignedNalUnitSize = mfx::align_value<int32_t>(size, 128);
+    AlignedNalUnitSize = mfx::align2_value<int32_t>(size, 128);
```

Comment:
Do we still need explicit `int32_t` here?

BTW, `size/AlignedNalUnitSize` are naturally unsigned, need to fix this at separate PR.

Reply:
yes, I will update